src/ssl.c

```diff
@@ -40246,6 +40707,15 @@ int wolfSSL_EC_KEY_LoadDer_ex(WOLFSSL_EC_KEY* key, const unsigned char* derBuf,
         ret = wc_EccPublicKeyDecode(derBuf, &idx, (ecc_key*)key->internal,
                                     derSz);
     }
+    if (ret < 0 && opt == WOLFSSL_EC_KEY_LOAD_PRIVATE) {
+        /* Might be in PKCS8 format so let's try */
+        idx = 0;
+        ret = ToTraditionalInline(derBuf, &idx, (word32)derSz);
+        if (ret > 0) {
+            ret = wc_EccPrivateKeyDecode(derBuf, &idx,
```

Comment:
The `wc_EccPrivateKeyDecode` handles PKCS8 format as well now. Do you need the `ToTraditionalInline` check?

Reply:
Good catch. Removed as its now redundant.